ObjectDetection/ObjectDetection/Inference/InferenceModule.mm

```diff
@@ -34,10 +33,13 @@ - (nullable instancetype)initWithFileAtPath:(NSString*)filePath {
 - (NSArray<NSNumber*>*)detectImage:(void*)imageBuffer {
     try {
         at::Tensor tensor = torch::from_blob(imageBuffer, { 1, 3, input_width, input_height }, at::kFloat);
-        torch::autograd::AutoGradMode guard(false);
```

Comment:
Why has this been removed?

Reply:
Same as Tao's answer above. It's replaced by `c10::InferenceMode guard;`